src/panel/context/Timeline.tsx

```diff
@@ -10,13 +10,20 @@ import React, {
 } from "react";
 import { scaleLinear } from "d3-scale";
 import { ParsedEvent } from "../types";
+import {
+  TimelineTooltip,
+  TooltipPosition
+} from "../pages/timeline/components/TimelineTooltip";
 import { DevtoolsContext } from "./Devtools";
 
 interface TimelineContextValue {
   events: Record<string, any>;
   setContainer: (e: HTMLDivElement) => void;
   getTimePosition: (t: number) => number;
   timelineLength: number;
+  setTooltipPosition: React.Dispatch<
```

Comment:
Its a good idea to keep state high in context but for tooltips, I don't think we'll need it as it's purely visual and specific to individual components.